project/settings.py

```
@@ -1,13 +1,22 @@
+import cloudinary
+import cloudinary.uploader
+from cloudinary.utils import cloudinary_url
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent
+# Configuration
+cloudinary.config(
+    cloud_name=config("CLOUDINARY_NAME"),
+    api_key=config("CLOUDINARY_API_KEY"),
+    api_secret=config("CLOUDINARY_API_SECRET"),
+    secure=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)